### ISSUE TITLE
Callout does not render if there is not space between > and [!calloutTitle]

### ISSUE THREAD

**Issue by Anthony-J-Cole** [MSzturc/obsidian-advanced-slides#267]
**Describe the bug**
Callouts are not rendered in slides if there is not a space between the > and [ while in obsidian it renders correctly. 

**Samples to Reproduce**
Create a Callout with no space between the > and [
Gets rendered as blockquote
**Expected behavior**
Callout renders correctly with title and symbol

**Comment by agentlibre** [comment]
I don't see it as a true issue since the space also works in Obsidian.

Just adapt the practice in Obsidian as it works for both?

**Comment by Anthony-J-Cole** [comment]
Agreed this is an extremely minor issue. Regardless, Obsidian renders the callout both with and without the space. In the slides not rendering it correctly without a space, it is not correctly addressing an intended behaviour and is therefore a bug.

From a users point of view who doesn’t know that it currently only works with a space, in Obsidian it is working correctly (them using not using a space), but the slide is rendering it as a block quote even though documentation says it supports callouts. The user would say this is a bug.

**Comment by EleyondFarli** [comment]
Spent a full day not understanding why callouts were not rendering, as I always write callouts in Obsidian without a space between > and [ . Then I stumbled on this. So this is affecting some users
